### PR TITLE
SQA tags for plane strain and radial displacement tests.

### PR DESCRIPTION
**Pull request by jasondhales**
#13634

### CODE REVIEW

**Review by gambka — CHANGES_REQUESTED** [review]

File: modules/tensor_mechanics/test/tests/generalized_plane_strain/tests [hunk 1]
```diff
@@ -1,52 +1,72 @@
...
- [../]
- [./generalized_plane_strain_scalar_vector]
-   type = 'Exodiff'
-   input = 'generalized_plane_strain_scalar_vector.i'
-   exodiff = 'generalized_plane_strain_scalar_vector_out.e'
-   prereq = 'generalized_plane_strain_squares'
-   abs_zero = 1e-7
-   allow_test_objects = true
- [../]
+    requirement = 'The system shall support a traditional plane strain mechanics solution'
```
[the remark this answers is not in the document]
Comment (jasondhales) [reply modules/tensor_mechanics/test/tests/generalized_plane_strain/tests]: As discussed, this language follows the example of other MOOSE tests.

File: modules/tensor_mechanics/test/tests/generalized_plane_strain/tests [hunk 2]
```diff
@@ -1,52 +1,72 @@
...
+    design = 'ComputePlaneSmallStrain.md'
+  [../]
+  [./plane_strain_prescribed]
+    type = 'Exodiff'
+    input = 'plane_strain_prescribed.i'
+    exodiff = 'plane_strain_prescribed_out.e'
+    custom_cmp = 'plane_strain.exodiff'
+    requirement = 'The system shall support a traditional plane strain mechanics solution where the out of plane strain is prescribed'
```
[the remark this answers is not in the document]
Comment (jasondhales) [reply modules/tensor_mechanics/test/tests/generalized_plane_strain/tests]: Fixed.

File: modules/tensor_mechanics/test/tests/radial_disp_aux/tests [hunk 3]
```diff
@@ -52,5 +69,7 @@
     cli_args = "AuxKernels/raddispaux/axis_vector='0 0 0'"
     expect_err = "axis_vector must have nonzero length"
     prereq = cylinder_3d_cartesian
+    design = 'RadialDisplacementCylinderAux.md'
+    requirement = 'The system shall report an error if "axis_vector" is has length of zero'
```
[the remark this answers is not in the document]
Comment (jasondhales) [reply modules/tensor_mechanics/test/tests/radial_disp_aux/tests]: Fixed.

**Review by gambka — APPROVED** [review]
Looks good to me.

**Review by permcody — APPROVED** [review]

**Pull request merged**